tests/NorthwindWebsite.Tests/Tests/CategoriesControllerTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NorthwindWebsite.Controllers;
+using NorthwindWebsite.Infrastructure.Entities;
+using NorthwindWebsite.Services.Interfaces;
+using NorthwindWebsite.Tests.Factories;
+
+namespace NorthwindWebsite.Tests.ControllersTests;
+
+public class CategoriesControllerTests
+{
+    private readonly Mock<ICategoryService> _categoryServiceMock = new();
+
+    private readonly CategoriesTestDataProvider _dataProvider = new();
+
+    [Fact]
+    public async Task IndexAction_VerifyCorrectViewAndModelAreReturned()
```

Comment:
For tests it's advised to add `Should` prefix before action you would like to perform in tests according to code conventions